OpenEdXMobile/src/test/java/org/edx/mobile/util/UrlUtilTest.java

```diff
@@ -0,0 +1,49 @@
+package org.edx.mobile.util;
+
+import org.edx.mobile.logger.Logger;
+import org.edx.mobile.test.BaseTestCase;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class UrlUtilTest extends BaseTestCase {
```

Comment:
There should be 1 more test  case for a utility method i.e. with following params

```
String baseURL = "http://www.fakex.com/course";
queryParams.put("search_query", "mobile linux");
queryParams.put("subject", "Architecture");
```